### ISSUE TITLE
FE | Profile | Accessibility | Mobile #

### ISSUE THREAD

**Issue by ericasauv09** [department-of-veterans-affairs/va.gov-team#49885]
## Background

The Accessibility team noted the following issue for screen reader users and the need to have content properly labeled:

> Within Profile Notification Settings, near the top of the content, the Veteran’s mobile number is listed, followed by a link to update their mobile phone number. The text content listed beforehand does not truly indicate what type of phone number is being read aloud by screen readers. Users may be able to guess that this is their mobile number, but the lack of specific callout text may create confusion.

Liz reviewed this with the accessibility folks, and the recommendation is to:

- Add a label in front of the piece of contact information to say what it is. We'll use a bold label in front of the piece of contact information. Mobile phone for mobile phones, and email address for email addresses.

- The links can be changed to say "update", but in an aria-label we should be more specific and say "mobile number" or "email address" for screenreaders on the link.

- Sketch files for [desktop](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c) and [mobile](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c/p/B5792AF8-87F2-496D-AE7A-DAD1448A6018/canvas) 

## Tasks
- [ ] Dev changes based on sketch files provided:
     - [ ] Change the mobile phone info on the Notification settings page to match the mock (includes re-ordering text, use of bold, change button to "update"). 
     - [ ] The links can be changed to say "update", but in an aria-label we should be more specific and say "mobile number" (or "email address" in future) for screenreaders on the link.
- [ ] Conduct Unit tests 
- [ ] Run E2E tests covering current code and regression
- [ ] Notify PM if this needs to be deployed to Staging behind feature flag
  

## Acceptance Criteria
- [ ] Complete all unit testing
- [ ] End-to-end tests showing 0 violations
- [ ] All axe checks pass
- [ ] Notify Product Manager that the change is ready to be validated on Staging
- [ ] Incorporate any changes resulting from Staging review & re-test & have re-validated
- [ ] Confirm PM has opened a new ticket to deploy changes to Production using feature flag, if needed

## Validation
- Who can validate this ticket? (FE, BE, Design, PM)?
- How can this work be validated?
- What updates need to be made (e.g. product outline, use cases, contact center guide)

## Associated Tickets
- #49589

**Comment by SKasimow** [comment]
HI - could you add the link(s) for the specific pages we want FE to update? The link Liz provided went to a section of screens and not sure which page(s) changed or what exactly changed.

**Comment by ericasauv09** [comment]
yea I saw that, maybe @andaleliz can weigh in on if there is a specific sketch file we should use? I thought these sketches covered different scenarios and that is why there is multiple, once Liz clarifies I can update the ticket

**Comment by andaleliz** [comment]
The devs can look at any screens on the [desktop](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c) or [mobile pages](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c/p/B5792AF8-87F2-496D-AE7A-DAD1448A6018/canvas) of this prototype to see the update.[ Here is a specific page if that's helpful](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c/a/52l205y).

**Comment by andaleliz** [comment]
Ah, I thought I had done that in [this comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/49589#issuecomment-1320511494), but I guess that isn't clear enough. I'll be more specific in the future.

Yes, that is the scope.

**Comment by SKasimow** [comment]
Thanks - I saw your earlier comment but think it was the link to multiple screens that threw me --- wanted to make sure I wasn't missing anything!

**Comment by SKasimow** [comment]
@andaleliz - we recently closed ticket [#48753 ](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/48753) where the "update" mobile number is supposed to be on it's own line. As Adam commented in this ticket:

> Here is what user 15 in staging looks like in mobile view
> 
> ![Screen Shot 2022-11-29 at 5.54.00 PM.png](https://images.zenhubusercontent.com/60ddebd504ce2452d0e0fa92/b6f07c80-2c0e-48a7-87e1-1f2d5cb24761)

(Note: they are no longer on a separate line and I've asked Adam about this).

My question is whether this ticket 49885 should be closed;  the mocks link to sketch files that have "update" on same line and not a separate line: [desktop](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c)

**Comment by andaleliz** [comment]
@SKasimow the designs are responsive, so the update link drops to it's own line as the screen gets smaller. Does that help? It looks like none of the tasks have been completed based on the check list in the description, so I do not think this ticket should be closed.

**Comment by SKasimow** [comment]
@andaleliz - yes thanks; I thought all screen sizes including desktop were going to have "update" on a second line.
Tasks/FE work was done as part of #48753 and staging and mocks are in synch - which is why I think this ticket can be closed as duplicative.
